src/utils/localRequire.js

```diff
@@ -1,16 +1,29 @@
 const {dirname} = require('path');
 const resolve = require('resolve');
+const install = require('./installPackage');
 
 const cache = new Map();
 
-module.exports = function(name, path) {
+async function localRequire(name, path, triedInstall = false) {
   let basedir = dirname(path);
   let key = basedir + ':' + name;
   let resolved = cache.get(key);
   if (!resolved) {
-    resolved = resolve.sync(name, {basedir});
+    try {
+      resolved = resolve.sync(name, {basedir});
+    } catch (e) {
+      if (e.code === 'MODULE_NOT_FOUND' && triedInstall === false) {
+        // TODO: Make this use logger
+        console.log(`INSTALLING ${name}...\n`);
+        await install(basedir, name);
```

Comment:
maybe we only want to do this for the top-level project, not for packages inside node_modules?

Reply:
Isn't this the responsibility of localRequire?

Reply:
Than wouldn't this break if a dependancy isn't use a base language?

Reply:
but some packages like request-promise don't install by default dependencies

Reply:
How do you mean @EduardoRFS ? This PR is just for auto installing missing parsers, through changing localRequire, ofcourse this can be used for way more things in the future